docker/thirdparties/docker-compose/hive/scripts/create_preinstalled_table.hql

```diff
@@ -31,6 +31,289 @@ TBLPROPERTIES (
 
 msck repair table partition_table;
 
+
+CREATE TABLE `delta_byte_array`(
+  `c_salutation` string,
+  `c_first_name` string,
+  `c_last_name` string,
+  `c_preferred_cust_flag` string,
+  `c_birth_country` string,
+  `c_login` string,
+  `c_email_address` string,
+  `c_last_review_date` string,
+  `c_customer_id` string
+  )
+ROW FORMAT SERDE
+  'org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe'
+STORED AS INPUTFORMAT
+  'org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat'
+OUTPUTFORMAT
+  'org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat'
+LOCATION
+  '/user/doris/preinstalled_data/different_types_parquet/delta_byte_array'
+TBLPROPERTIES (
+  'transient_lastDdlTime'='1661955829');
+
+
+CREATE TABLE `delta_length_byte_array`(
+  `FRUIT` string
+  )
+ROW FORMAT SERDE
+  'org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe'
+STORED AS INPUTFORMAT
+  'org.apache.hadoop.hive.ql.io.parquet.MapredParquetInputFormat'
+OUTPUTFORMAT
+  'org.apache.hadoop.hive.ql.io.parquet.MapredParquetOutputFormat'
+LOCATION
+  '/user/doris/preinstalled_data/different_types_parquet/delta_length_byte_array'
+TBLPROPERTIES (
+  'transient_lastDdlTime'='1661955829');
+
+msck repair table delta_length_byte_array;
+
+CREATE EXTERNAL TABLE `delta_binary_packed`(
+  bitwidth0 bigint,
+  bitwidth1 bigint,
+  bitwidth2 bigint,
+  bitwidth3 bigint,
+  bitwidth4 bigint,
+  bitwidth5 bigint,
+  bitwidth6 bigint,
+  bitwidth7 bigint,
+  bitwidth8 bigint,
+  bitwidth9 bigint,
+  bitwidth10 bigint,
+  bitwidth11 bigint,
+  bitwidth12 bigint,
+  bitwidth13 bigint,
+  bitwidth14 bigint,
+  bitwidth15 bigint,
+  bitwidth16 bigint,
+  bitwidth17 bigint,
+  bitwidth18 bigint,
+  bitwidth19 bigint,
+  bitwidth20 bigint,
+  bitwidth21 bigint,
+  bitwidth22 bigint,
+  bitwidth23 bigint,
+  bitwidth24 bigint,
+  bitwidth25 bigint,
+  bitwidth26 bigint,
+  bitwidth27 bigint,
+  bitwidth28 bigint,
+  bitwidth29 bigint,
+  bitwidth30 bigint,
+  bitwidth31 bigint,
+  bitwidth32 bigint,
+  bitwidth33 bigint,
+  bitwidth34 bigint,
+  bitwidth35 bigint,
+  bitwidth36 bigint,
+  bitwidth37 bigint,
+  bitwidth38 bigint,
+  bitwidth39 bigint,
+  bitwidth40 bigint,
+  bitwidth41 bigint,
+  bitwidth42 bigint,
+  bitwidth43 bigint,
+  bitwidth44 bigint,
+  bitwidth45 bigint,
+  bitwidth46 bigint,
+  bitwidth47 bigint,
+  bitwidth48 bigint,
+  bitwidth49 bigint,
+  bitwidth50 bigint,
+  bitwidth51 bigint,
+  bitwidth52 bigint,
+  bitwidth53 bigint,
+  bitwidth54 bigint,
+  bitwidth55 bigint,
+  bitwidth56 bigint,
+  bitwidth57 bigint,
+  bitwidth58 bigint,
+  bitwidth59 bigint,
+  bitwidth60 bigint,
+  bitwidth61 bigint,
+  bitwidth62 bigint,
+  bitwidth63 bigint,
+  bitwidth64 bigint,
+  int_value  int
+  )
+LOCATION
+  '/user/doris/preinstalled_data/different_types_parquet/delta_binary_packed'
```

Comment:
Need to specify the file format, for example, `stored as parquet`. Otherwise, hive will try to decode it as text format.